notebook/images/ccg-workshop/runAnalysis.1.8.sh

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env bash
```

Comment:
this doesn't get run as part of the build, right? It's just copied?

```
$ rg "runAnalysis" ..

../notebook/images/ccg-workshop/Dockerfile
165:COPY runAnalysis.1.8.sh WEScourse/
```

Reply:
we could alternatively just delete that line

Reply:
correct, the copy fails.